backend/fregepoc/repositories/tasks.py

```diff
@@ -104,6 +107,8 @@ def process_repo_task(repo_pk):
 
     for repo_file in repo_files:
         analyze_file_task.apply_async(args=(repo_file.pk,))
+
+    repo_obj.close()
```

Comment:
I suggest using `contextlib.closing` context manager instead: https://docs.python.org/3/library/contextlib.html#contextlib.closing instead as it basically closes objects in a `final` statement ensuring us that the resource will have gotten closed even if some kind of error has occurred.